<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WireConvertForm</class>
 <widget class="QWidget" name="WireConvertForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>320</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>320</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>线束表转化</string>
  </property>
  <widget class="QLabel" name="label01">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>EPLAN文件：</string>
   </property>
   <property name="buddy">
    <cstring>lineEIFile</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEIFile">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>10</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnIFile">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbStart">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label02">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>数据起始行:</string>
   </property>
   <property name="buddy">
    <cstring>spbStart</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label03">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>结束行(0所有行):</string>
   </property>
   <property name="buddy">
    <cstring>spbEnd</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbEnd">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPause">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QLabel" name="label06">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>线束BOM</string>
   </property>
   <property name="buddy">
    <cstring>lineEOFile</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEOFile">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>90</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
  </widget>
  <widget class="QToolButton" name="btnOFile">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbType">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>陆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEIFile</tabstop>
  <tabstop>btnIFile</tabstop>
  <tabstop>spbStart</tabstop>
  <tabstop>spbEnd</tabstop>
  <tabstop>lineEOFile</tabstop>
  <tabstop>btnOFile</tabstop>
  <tabstop>btnStart</tabstop>
  <tabstop>btnPause</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
